Added Popup Component and Bug Fixes

nickci2002 opened the pull request:
Creates a popup component to be used anywhere in the code. Also, fixed a bug in form validation where the program would print "Invalid email!" when logging in.

nickci2002 committed:
Temp Work

Is this OK?

nickci2002 committed:
Completed Form Validation

nickci2002 committed:
Fixed bug

TODO: Fix error display

banmeetkaur1 committed:
Merge branch 'main' into banmeet_branch

nickci2002 committed:
Merge branch 'main' into origin/nick_branch

nickci2002 committed:
ssss

nickci2002 committed:
Update login.tsx

nickci2002 committed:
Popup Component Completed and Bug Fixes

Completed the Popup Component and fixed the FormValidation component from printing "Invalid Email" when no email was entered.

FarWrong commented:
merge recent changes into ur branch and then open another pull request. Other then that, this looks pretty good. Good use of themes

nickci2002 committed:
Merge branch 'main' into origin/nick_branch

nickci2002 committed:
Delete shop.tsx

nickci2002 committed:
Added popup component to textbox

nickci2002 committed:
Merge branch 'main' into origin/nick_branch

nickci2002 committed:
CSS Alteration

Moved CSS styling to style.tsx. theme.tsx still contains the colors for themes.

banmeetkaur1 committed:
profile screen changes, prettify home screen + data entry screen

nickci2002 committed:
Merge branch 'banmeet_branch' into origin/nick_branch

nickci2002 commented:
I've fixed everything and it's been a week, can you merge it?

nickci2002 committed:
Fixed Login Screen to Use Premade Components
